src/commands/play.ts

```diff
@@ -50,7 +50,11 @@ async function play(guild: Guild, queue: SongQueue) {
   }
 }
 
-export async function executePlay(interaction, queue, songRequest = null) {
+export async function executePlay(
+  interaction: ButtonInteraction,
```

Comment:
I would say it's not always ButtonInteraction. 
For now this functions is called directly after the button clicked of the search command, but what if we also want to implement the play command (without search) ? I think it would be better to use some thing like Interaction, then check if it's instance of ButtonInteraction or MessageInteraction...

Reply:
so ButtonInteraction | CommandInteraction ?

Reply:
Need to think about this later, accept for now